package.json

```diff
@@ -17,7 +17,7 @@
         "migrate": "NODE_TLS_REJECT_UNAUTHORIZED=0 node-pg-migrate",
         "dev": "nodemon",
         "start": "ts-node ./src/index.ts",
-        "start:prd": "npm run migrate up && node ./dist/src/index.js",
+        "start:prd": "node migrateUpWithWrapper.mjs && node ./dist/src/index.js",
```

Comment:
no problem. 

For context, the change means: call that script the script `migrateUpWithWrapper.mjs`. This script, in turn, calls the script `npm run migrate` (see it in package.json). I have added that wrapper so that errors from `npm run migrate up` are parsed and if need be, modify the logs in order to avoid leaking info.